Index: src/uncat.check
===================================================================
RCS file: /var/lib/cvs/socklog/src/uncat.check,v
retrieving revision 1.1
diff -u -r1.1 uncat.check
--- src/uncat.check	28 Feb 2004 15:54:24 -0000	1.1
+++ src/uncat.check	29 Feb 2004 16:49:56 -0000
@@ -7,7 +7,7 @@
 
 ( echo foo; sleep 1; echo bar ) |uncat -vot14 cat
 echo $?
-( echo foo; sleep 2; echo bar ) |uncat -vot1 cat
+( echo foo; sleep 3; echo bar ) |uncat -vot2 cat
 echo $?
 ( echo foo; echo bar; echo baz ) |uncat -vos4 cat
 echo $?
